config/build.yml

```diff
@@ -113,6 +113,8 @@ phpcbf:
 # @todo Move to subkey of tests.
 phpunit:
   - path: '${repo.root}/tests/phpunit'
+    class: 'ExampleTest'
```

Comment:
By default, BLT does not place this file in the project repository. You must first run `blt examples:init` in order to generate the file.

So, I would expect this change to cause `blt tests:phpunit` to fail out of the box.